browser/ui/brave_pages.cc

```diff
@@ -5,6 +5,7 @@
 
 #include "brave/browser/ui/brave_pages.h"
 
+#include "brave/browser/ipfs/ipfs_tab_helper.h"
```

Comment:
This header itself is guarded by the build flag, so you'll have to have buildflag when using it too.